tests/test_containers.py

```diff
@@ -0,0 +1,158 @@
+import gzip
+import logging
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric.rsa import (
+        generate_private_key, RSAPrivateKey)
+import docker
+import pytest
+import time
+
+from proof_of_concept.components.ddm_site import Site
+from proof_of_concept.components.registry_client import RegistryClient
+from proof_of_concept.definitions.assets import ComputeAsset, DataAsset
+from proof_of_concept.definitions.registry import (
+        PartyDescription, SiteDescription)
+from proof_of_concept.definitions.workflows import Job, Workflow, WorkflowStep
+from proof_of_concept.policy.rules import (
+        MayAccess, ResultOfComputeIn, ResultOfDataIn)
+from proof_of_concept.rest.ddm_site import SiteRestApi, SiteServer
+
+
+logger = logging.getLogger(__file__)
+
+
+@pytest.fixture
+def dcli():
+    """Docker client object."""
+    return docker.from_env()
+
+
+@pytest.fixture
+def docker_dir(test_dir):
+    """Path to the tests/docker directory."""
+    return test_dir / 'docker'
```

Comment:
I usually use constants for filepaths such as this. It is explicit and less verbose than fixtures. What is the advantage of using fixtures for this?

Reply:
Mainly symmetry with pytest's `tmpdir` fixture I guess, and my general distaste for globals :-)